include/common/datatypes/DataSetOps-inl.h

```diff
@@ -0,0 +1,183 @@
+/* Copyright (c) 2020 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+#pragma once
+
+#include <thrift/lib/cpp2/GeneratedCodeHelper.h>
+#include <thrift/lib/cpp2/gen/module_types_tcc.h>
+#include <thrift/lib/cpp2/protocol/ProtocolReaderStructReadState.h>
+
+#include "common/datatypes/CommonCpp2Ops.h"
+#include "common/datatypes/DataSet.h"
+#include "common/utils/utils.h"
+
+namespace apache {
+namespace thrift {
+
+/**************************************
+ *
+ * Ops for class DataSet
+ *
+ *************************************/
+namespace detail {
+
+template <>
+struct TccStructTraits<nebula::DataSet> {
+    static void translateFieldName(MAYBE_UNUSED folly::StringPiece _fname,
+                                   MAYBE_UNUSED int16_t& fid,
+                                   MAYBE_UNUSED apache::thrift::protocol::TType& _ftype) {
+        if (_fname == "column_names") {
+            fid = 1;
+            _ftype = apache::thrift::protocol::T_LIST;
+        } else if (_fname == "rows") {
+            fid = 2;
+            _ftype = apache::thrift::protocol::T_LIST;
+        }
```

Comment:
Why does nebula-cpp need these Ops-in.h files?

Reply:
For more compatible we copy the server implementation.